installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc

```diff
@@ -39,10 +39,15 @@ include::modules/installing-ocp-agent-ibm-z.adoc[leveloffset=+1]
 // Adding {ibm-z-title} agents with z/VM
 include::modules/installing-ocp-agent-ibm-z-zvm.adoc[leveloffset=+2]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on {ibm-z-title} and {ibm-linuxone-title}]
+
 // Adding {ibm-z-name} agents with {op-system-base} KVM
 include::modules/installing-ocp-agent-ibm-z-kvm.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
-[id="additional-resources_prepare-pxe-assets-agent"]
+[id="additional-resources_prepare-pxe-assets-agent_{context}"]
 == Additional resources
 * See xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc#installing-with-agent-based-installer[Installing an {product-title} cluster with the Agent-based Installer] to learn about more configurations available with the Agent-based Installer.
```

Comment:
While here, consider updating the lead-in text for thie section to follow [this guideline](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute).

Reply:
This is unfortunately an instance that would benefit from having that extra context to tell users what they can find on that other page. I'll consider either removing the non-link text or moving this link elsewhere in the assembly, such as the intro paragraph